src/Components/RemediationsPage/__tests__/RemediationsPage.test.js

```diff
@@ -0,0 +1,72 @@
+import '@testing-library/jest-dom';
+
+import React from 'react';
+import axios from 'axios';
+import { usePermissions } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
+import { render, screen, waitFor } from '@testing-library/react';
+
+import RemediationsPage from '../RemediationsPage';
+
+jest.mock('@redhat-cloud-services/frontend-components-utilities/RBACHook', () => ({
+  usePermissions: jest.fn(),
+}));
+jest.mock('axios');
+
+describe('RemediationsPage', () => {
+  beforeEach(() => {
+    usePermissions.mockImplementation(() => ({ hasAccess: true, isLoading: false }));
+  });
+
+  it('should render table with recommendations', async () => {
```

Comment:
Shouldn't this whole file use the remediation word isntead of recommendation? I assume it was just copied from recommendations :sweat_smile: